src/utils/filter.ts

```diff
@@ -1,28 +1,30 @@
 import { TokenDetails } from 'types'
 import { DetailedAuctionElement, Trade } from 'api/exchange/ExchangeApi'
 import { computeMarketProp } from './display'
+import { web3 } from 'api'
 
 export function checkTokenAgainstSearch(token: TokenDetails | null, searchText: string): boolean {
   if (!token) return false
   return (
     token?.symbol?.toLowerCase().includes(searchText) ||
     token?.name?.toLowerCase().includes(searchText) ||
-    token?.address.toLowerCase().includes(searchText)
+    (web3.utils.isAddress(searchText) && token?.address.toLowerCase().includes(searchText))
```

Comment:
nit: `web3.utils.isAddress(searchText)` is evaluated for every row two times. Should we pre-calculate it and just pass the flag (or use a different function depending on it)?

Reply:
but im checking both tokens, `buyToken` and sell, so it needs to eval twice, no?

Reply:
see what u mean, made changes

Reply:
`isAddress` check is checksum sensitive so if you pass lowercase to it, will return `false` always

Reply:
@Velenir it won't return `false` unless the mixed cases are wrong. if all lowercase or all uppercase it skips checksum check

Reply:
https://web3js.readthedocs.io/en/v1.2.7/web3-utils.html#isaddress